README.md

```diff
@@ -36,33 +36,33 @@ $ docker push <username>/shopping-cart
 
 ## Deploying to Lightbend Cloudstate
 
-The following steps use `csctl` to deploy the application to [Lightbend Cloudstate](https://docs.lbcs.io/).
+The following steps use `akkasls` to deploy the application to [Lightbend Cloudstate](https://docs.lbcs.io/).
```

Comment:
We should be pointing at the docs at docs.cloudstate.com. However, the docs will soon be published on developer.lightbend.com/docs, so it will have to be changed again later. I suspect we will have docs.cloudstate.com redirect to developer.lightbend.com, so it is okay to change it to docs.cloudstate.com for now.